tests/src/test/scala/cats/tests/KleisliTests.scala

```diff
@@ -15,8 +15,25 @@ class KleisliTests extends CatsSuite {
   implicit def kleisliEq[F[_], A, B](implicit A: Arbitrary[A], FB: Eq[F[B]]): Eq[Kleisli[F, A, B]] =
     Eq.by[Kleisli[F, A, B], A => F[B]](_.run)
 
+  implicit val xorT = XorT.xorTEq[Kleisli[Option, Int, ?], Unit, Int]
+
+  implicit val optionApplicativeError : ApplicativeError[Option, Unit] = new ApplicativeError[Option, Unit] {
```

Comment:
I think it could give users an option (hah) to plug in `Option` if working with a `MonadError` API (I'm guessing `Option` can implement `MonadError`, not just `ApplicativeError`) when they don't care about the errors